[EDU-5108] feat: add CRUD for record and table (#1243)
aziontech/docs: src/content/docs/en/pages/store-journey/sql/query-rows.mdx

---
title: How to retrieve data from tables in Edge SQL

> description: Learn how to efficiently list and retrieve data from Edge SQL databases using the Azion REST API. This comprehensive guide provides step-by-step instructions for querying tables and optimizing your database management.

Reviewer: ```suggestion
description: This comprehensive guide provides step-by-step instructions to efficiently list and retrieve data from Edge SQL databases using the Azion REST API. 
```

Reviewer: < 160 chars

meta_tags: 'Edge SQL, SQL query, list data, retrieve data, database management, Azion REST API, SQL tutorial, data retrieval, database optimization, querying tables'
namespace: docs_store_journey_sql_create_table
permalink: /documentation/products/store/sql/retrieve-data-edge-sql/
---

import LinkButton from 'azion-webkit/linkbutton'

> Once you've created your Edge SQL database, you can begin creating tables and inserting data into them.

Reviewer: ```suggestion
Once you've created your Edge SQL database, you can begin retrieving data from them.
```
